### PR TITLE
Fee market optimization

### PR DESCRIPTION
**Pull request by JayJay1024**

### CODE REVIEW

File: src/components/feemarket/OrderDetail.tsx [hunk 1]
```diff
@@ -1,17 +1,39 @@
...
+  RelayerRole,
+  SubqlOrderStatus,
+  OrderStatus,
+} from '../../model';
 import { useApi } from '../../hooks';
 import { SubscanLink } from '../widget/SubscanLink';
 import { fromWei, prettyNumber } from '../../utils';
 import { AccountName } from '../widget/account/AccountName';
 
+const getPrioritySlot = (t: TFunction, confirmedSlotIndex?: number | null): string => {
```
Comment (sxlwar) [inline src/components/feemarket/OrderDetail.tsx]: Do not pass 't' function; 

Better to use enum here

File: src/components/feemarket/RelayerDetail.tsx [hunk 2]
```diff
@@ -280,55 +279,78 @@ export const RelayerDetail = ({
...
-
-      setDataSource(
-        orders
-          .map((order) => {
-            const role = new Set<RelayerRole>();
-
-            if (order.assignedRelayers.some((item) => item === relayerAddress)) {
+      const { assignedOrders, deliveredOrders, confirmedOrders, slashs } = relayerOrdersData.relayerEntity;
+
+      let ds =
```
Comment (sxlwar) [inline src/components/feemarket/RelayerDetail.tsx]: 1. use meaningful naming;  what does **ds** mean?
2. DRY

File: src/hooks/pollIntervalQuery.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import type {
+  DocumentNode,
+  OperationVariables,
+  TypedDocumentNode,
+  QueryHookOptions,
+  QueryResult,
+} from '@apollo/client';
+
+import { LONG_LONG_DURATION } from '../config';
+
```
Comment (sxlwar) [inline src/hooks/pollIntervalQuery.tsx:11]: ts warning

File: src/model/feemarket.ts [hunk 4]
```diff
@@ -22,6 +22,42 @@ export enum SegmentedType {
...
+  RELAYERS = 'relayers',
+  OREDERS = 'oreders',
+}
+
+export enum RelayerRole {
+  ASSIGNED = 'Assigned Relayer',
+  DELIVERY = 'Delivery Relayer',
+  CONFIRMED = 'Confirmed Relayer',
+}
+
```
Comment (sxlwar) [inline src/model/feemarket.ts:36]: Unify enum naming style

**Review by sxlwar — CHANGES_REQUESTED** [review]

**Review by ZolaxZola — APPROVED** [review]

**Review by sxlwar — APPROVED** [review]

**Pull request merged**